### PR TITLE
Add comments to the Clusterfile

### PR DESCRIPTION
**Pull request by ly-wjj**
### Describe what this PR does / why we need it
Increase the readability of documents

### Does this pull request fix one issue?
No

### Describe how you did it
Added a comment in the configuration example of Clusterfile in the README.md file.
Explains how to use ssh configuration

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,9 +135,13 @@ spec:
   image: registry.cn-qingdao.aliyuncs.com/sealer-io/kubernetes:v1.19.9
   provider: BAREMETAL
   ssh:
+    ## ssh的登录密码，如果使用的密钥登录则无需设置
```
Comment (fanux) [inline README.md:138]: This is English document, may be you could using English? 
Chinese docs are in the docs/README_zh.md

Comment (ly-wjj) [reply README.md]: I have changed the configuration as required

**Comment by cuisongliu** [comment]
@ly-wjj  please use English for your pr (pr title and commit info).

**Comment by cuisongliu** [comment]
merge commit , 12 times commit is too more. can you merge commit to 1 times?

**Comment by ly-wjj** [comment]
> merge commit , 12 times commit is too more. can you merge commit to 1 times?

ok, I have done

**Pull request merged**